core/src/repl/query_print.rs

```diff
@@ -1,11 +1,10 @@
 //! Rendering of the results of a metadata query.
-use crate::{
-    identifier::{Ident, LocIdent},
-    pretty::PrettyPrintCap,
```

Comment:
This is what is failing the CI - the trait `pretty::PrettyPrintCap` has been added recently (maybe before you  branched?) and need to be kept included here.